tools/custom-checks/run-custom-checks.ts

```diff
@@ -6,19 +6,11 @@ import { checkSiblingDepVersionConsistency } from "./check-sibling-dep-version-c
 export async function runCustomChecks(
   argv: string[],
   env: NodeJS.ProcessEnv,
-  version: string,
```

Comment:
@Leeyoungone I wouldn't remove the version parameter itself because later on we might again find ourselves in a situation where we need to do some custom logic based on it (let's say Node v18 comes out and something doesn't work on it or something stops working on Node v14 which right now is perfectly fine, etc.)
If you removed it because the linter complained that it's not being used for anything then my trick for that usually is to just log it out so that a) we provide more information in the logs and b) the linter chills out

Reply:
Sounds good! Thank you for the cool trick 👓 !